### PR TITLE
License  headers and private library

### PR DESCRIPTION
**Pull request by agaida**
* make the included library private
* add missed license headers

### CODE REVIEW

**Comment by agaida** [comment]
@luis-pereira can you please have a look on the private .so patch?

i will rework the license headers for the 2 remaining files according to the mailing list.

**Comment by agaida** [comment]
@luis-pereira, @paulolieuthier, @jleclanche ping

**Comment by paulolieuthier** [comment]
AFAIK, the discussion in the ML has not settled.

**Comment by agaida** [comment]
In my not so humble opinion - we can change this later, but i need some headers merged - without the headers i don't get the package into debian :)

**Comment by paulolieuthier** [comment]
Then go ahead, I say it's GTM.

**Comment by jleclanche** [comment]
Yeah this is fine for now.

File: liblxqt-config-cursor/CMakeLists.txt [hunk 1]
```diff
@@ -80,7 +80,7 @@ target_link_libraries(lxqt-config-cursor
 
 install(TARGETS
     lxqt-config-cursor
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}"
```
Comment (luis-pereira) [inline liblxqt-config-cursor/CMakeLists.txt]: @agaida Sorry for the delay. Real life. 
This causes an  error while loading shared libraries. The dynamic linker (default configuration) will not find the library there.
Filled an issue: https://github.com/lxde/lxqt/issues/793

**Pull request merged**